integrations/salesforce/integration.go

```diff
@@ -69,6 +70,18 @@ func (h handler) reopenExistingPubSubConnections(ctx context.Context) {
 		instanceURL := data.GetValue(instanceURLVar)
 		orgID := data.GetValue(orgIDVar)
 
-		h.subscribe(instanceURL, orgID, cid)
+		var clientID string
+		if common.ReadAuthType(data) == integrations.OAuthPrivate {
+			clientID = data.GetValue(clientIDVar)
+		} else {
```

Comment:
This code has the exact same effect, without `else`
```suggestion
		clientID := data.GetValue(clientIDVar)
		if common.ReadAuthType(data) == integrations.OAuthDefault {
```